msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationConfigurationTest.kt

```diff
@@ -290,4 +290,97 @@ class NativeAuthPublicClientApplicationConfigurationTest {
         spyConfig.setChallengeTypes(listOf("oob", "oob", "password", "redirect", "redirect", "redirect"))
         spyConfig.validateConfiguration()
     }
+
+    // Capabilities are optional, so no exception should be thrown
+    @Test
+    fun testMissingCapabilities() {
+        val config = NativeAuthPublicClientApplicationConfiguration()
+        config.clientId = clientId
+        config.accountMode = AccountMode.SINGLE
+        val spyConfig = spy(config)
+        whenever(spyConfig.authorities).thenReturn(listOf(NativeAuthCIAMAuthority(ciamAuthority, clientId)))
+        whenever(spyConfig.defaultAuthority).thenReturn(NativeAuthCIAMAuthority(ciamAuthority, clientId))
+        whenever(spyConfig.isSharedDevice).thenReturn(false)
+        whenever(spyConfig.useBroker).thenReturn(false)
+        whenever(spyConfig.getCapabilities()).thenReturn(emptyList())
+        spyConfig.validateConfiguration()
+    }
+
+    @Test
```

Comment:
nit: you can use
```suggestion
    @Test(expected = MsalClientException::class)
```
To clarify that this test expects an exception is thrown. 
Advantages are:
- Clarity: reader can easily understand if and which exception is expected
- Shorter code: no need to write try/catch block

Reply:
Good suggestion. But my preference is to keep the try-catch structure with reference to other tests, as this allows for simultaneous validation of the `NATIVE_AUTH_INVALID_CAPABILITY_ERROR_CODE` inside `MSAL Exception`